docs/public/_redirects

```diff
@@ -11,12 +11,31 @@ https://next.material-ui.com/* https://mui.com/:splat 301!
 
 # Support multiple domains
 https://material-ui.dev/* https://mui.com/:splat 301!
-https://material-ui.com/* https://mui.com/:splat 301!
 https://v0.material-ui.com/* https://v0.mui.com/:splat 301!
 https://v1.material-ui.com/* https://v1.mui.com/:splat 301!
 https://v3.material-ui.com/* https://v3.mui.com/:splat 301!
 https://v4.material-ui.com/* https://v4.mui.com/:splat 301!
 
+# Redirect all except the store
+https://material-ui.com https://mui.com 301!
+https://material-ui.com/api-docs/* https://mui.com/api-docs/:splat 301!
+https://material-ui.com/api/* https://mui.com/api/:splat 301!
+https://material-ui.com/blog/* https://mui.com/blog/:splat 301!
+https://material-ui.com/company/* https://mui.com/company/:splat 301!
+https://material-ui.com/components/* https://mui.com/components/:splat 301!
+https://material-ui.com/customization/* https://mui.com/customization/:splat 301!
+https://material-ui.com/discover-more/* https://mui.com/discover-more/:splat 301!
+https://material-ui.com/getting-started/* https://mui.com/getting-started/:splat 301!
+https://material-ui.com/guides/* https://mui.com/guides/:splat 301!
+https://material-ui.com/performance/* https://mui.com/performance/:splat 301!
+https://material-ui.com/premium-themes/* https://mui.com/premium-themes/:splat 301!
```

Comment:
This redirects currently to the store,we should handle it the same as the store.